src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -55,44 +55,44 @@
+        {4, channel_messages_delivered_ack_total, counter, "Total number of messages delivered to consumers in manual acknowledgement mode"},
+        {5, channel_messages_delivered_total, counter, "Total number of messages delivered to consumers in automatic acknowledgement mode"},
+        {6, channel_messages_redelivered_total, counter, "Total number of messages redelivered to consumers"},
+        {7, channel_messages_acked_total, counter, "Total number of messages acknowledged by consumers"},
+        {8, channel_get_empty_total, counter, "Total number of times basic.get operations fetched no message"}
     ]},
 
     {connection_churn_metrics, [
-        {2, connection_open, counter, "Connections opened"},
-        {3, connection_close, counter, "Connections closed or terminated"},
-        {4, channel_open, counter, "Channels opened"},
-        {5, channel_close, counter, "Channels closed"},
-        {6, queue_declare, counter, "Queues declared"},
-        {7, queue_create, counter, "Queues created"},
-        {8, queue_delete, counter, "Queues deleted"}
+        {2, connection_open_total, counter, "Total number of connections opened"},
+        {3, connection_close_total, counter, "Total number of connections closed or terminated"},
+        {4, channel_open_total, counter, "Total number of channels opened"},
+        {5, channel_close_total, counter, "Total number of channels closed"},
+        {6, queue_declare_total, counter, "Total number of queues declared"},
+        {7, queue_create_total, counter, "Total number of queues created"},
+        {8, queue_delete_total, counter, "Total number of queues deleted"}
     ]},
 
     {connection_coarse_metrics, [
-        {2, connection_bytes_in, counter, "Bytes received on a connection"},
-        {3, connection_bytes_out, counter, "Bytes sent on a connection"},
-        {4, erlang_process_reductions, counter, "Erlang process reductions"}
+        {2, connection_bytes_in_total, counter, "Total number of bytes received on a connection"},
+        {3, connection_bytes_out_total, counter, "Total number of bytes sent on a connection"},
+        {4, erlang_process_reductions_total, counter, "Total number of Erlang process reductions"}
     ]},
 
     {connection_metrics, [
-        {2, connection_packets_in, counter, "Packets received on a connection", recv_cnt},
-        {2, connection_packets_out, counter, "Packets sent on a connection", send_cnt},
-        {2, connection_packets_pending, counter, "Packets waiting to be sent on a connection", send_pend},
+        {2, connection_packets_in_total, counter, "Total number of packets received on a connection", recv_cnt},
+        {2, connection_packets_out_total, counter, "Total number of packets sent on a connection", send_cnt},
+        {2, connection_packets_pending_total, counter, "Total number of packets waiting to be sent on a connection", send_pend},
```

Comment:
This sounds like a gauge

Reply:
Why gauge?

In this case I care about how fast these are increasing, not the absolute value.

Reply:
The number of items in a queue is a gauge, as it can increase and decrease and you care about the absolute value.

Reply:
Which metric are you referring to? I see `connection_packets_*` in the highlight above.

Reply:
connection_packets_pending_total